### DISCUSSION TITLE
Super slow query making my web app freeze in Phoenix

### DISCUSSION THREAD

**Discussion by nikhilbobb** [planetscale/discussion#253] [Bug Reports] [answered]
Hi - Not sure if this query is coming from my app or from planetscale but it is killing my performance. I can't find it doing a grep of all the libraries I'm using. It is taking 20-30 seconds.

Any ideas?

select get_lock(:vtg1, -:vtg2) from dual

**Comment by nikhilbobb** [comment, marked as answer]
solved by turning off the migration_lock in my ecto configuration. Noting for anyone who has the issue in the future.

**Reply by pepicrft** [reply]
I ran into the following error trying to run migrations in production:

```
** (MatchError) no match of right hand side value: {:error, %MyXQL.Error{connection_id: 2345279677, message: "in autocommit mode, transactionID should be zero but was: 1674845486107521990", mysql: %{code: 1105, name: nil}, statement: "SELECT RELEASE_LOCK('ecto_Gesttalt.Repo')"
```
And disabling the lock fixed the issue for me too 🎉
